### PR TITLE
Add PHP 7.1 and extension information

### PR DESCRIPTION
**Pull request by Crell**
Doc updates related to PHP 7.1.  Also includes clarification that we have sockets built in, and now ext-event and pthreads on 7.1.

### CODE REVIEW

File: src/languages/php.md [hunk 1]
```diff
@@ -134,8 +134,40 @@ This is the complete list of extensions that can be enabled:
 > **note**
 > You can check out the output of `ls /etc/php5/mods-available` to
 > see the up-to-date complete list of extensions after you SSH into
-> your environment. For PHP 7, use `ls /etc/php/mods-available`.
+> your environment. For PHP 7, use `ls /etc/php7.0//mods-available`.
```
Comment (pjcdawkins) [inline src/languages/php.md]: extra `/`

File: src/languages/php.md [hunk 2]
```diff
@@ -134,8 +134,40 @@ This is the complete list of extensions that can be enabled:
...
+> For PHP 7.1, use `ls /etc/php7.1-zts/mods/available`.
 
+## Alternate start commands
+
+PHP is most commonly run in a CGI mode, using PHP-FPM.  If you are not sure how PHP is configured to start then that is what is happening.  However, you can also start alternative processes if desired, such as if you're running an Async PHP daemon, a thread-based worker process, etc.  To do so, simply specify an alternative start command in `platform.app.yaml`, similar to the following:
```
Comment (pjcdawkins) [inline src/languages/php.md]: The sentence "If you are not sure" is probably unnecessary

File: src/languages/php.md [hunk 3]
```diff
@@ -51,6 +54,7 @@ The following extensions are enabled by default:
 * mysqlnd
 * pdo_mysql
 * pdo_sqlite
+* sockets (5.6 and later)
```
Comment (damz) [inline src/languages/php.md]: It's 7.0 and later, no?

File: src/languages/php.md [hunk 4]
```diff
@@ -62,73 +66,107 @@ You can disable those by adding them to the `disabled_extensions` list.
...
 
 > **note**
 > You can check out the output of `ls /etc/php5/mods-available` to
 > see the up-to-date complete list of extensions after you SSH into
-> your environment. For PHP 7, use `ls /etc/php/mods-available`.
+> your environment. For PHP 7, use `ls /etc/php/*/mods-available`.
+
+## Alternate start commands
+
+PHP is most commonly run in a CGI mode, using PHP-FPM.  That is the default on Platform.sh.  However, you can also start alternative processes if desired, such as if you're running an Async PHP daemon, a thread-based worker process, etc.  To do so, simply specify an alternative start command in `platform.app.yaml`, similar to the following:
```
Comment (ralt) [inline src/languages/php.md]: Is FastCGI mode != CGI mode?

Comment (Crell) [inline src/languages/php.md]: I'm being generic here, to refer to CGI in the abstract (aka, the web as if it were 1991).  PHP userspace can't really tell the difference between mod_php and FastCGI.  It definitely knows the difference between that and CLI, however.

**Pull request merged**